hack/ci/ensure-image.sh

```diff
@@ -13,19 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 set -e
 
-default_cluster_name="operator-test"
+source ./hack/env.sh
 
-if [[ -z ${CLUSTER_NAME} ]]; then
-    CLUSTER_NAME=$default_cluster_name
-fi
+version=$(getOperatorVersion)
 
-source ./hack/env.sh
-docker images
-echo "---> Loading Operator Image into Kind"
-kind load docker-image quay.io/kiegroup/kogito-cloud-operator:"$(getOperatorVersion)" --name ${CLUSTER_NAME}
+echo "=====> tagging image"
+docker tag quay.io/kiegroup/kogito-cloud-operator:${version} kind-registry:5000/kiegroup/kogito-cloud-operator
```

Comment:
shouldn't we use `IMAGE` env instead as the `Makefile` is doing ?
also shouldn't we add a task in the `Makefile` to do it ?

Reply:
Changed, and added make target `make olm-tests`